### PR TITLE
docs: remove $emdash text on lifeycycle hooks

### PR DESCRIPTION
**Pull request by trungvose**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
I read https://angular.io/guide/lifecycle-hooks this morning and see the text `$emdash`. It should not be there.

Issue Number: N/A

## What is the new behavior?

Remove the text `$emdash`

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by trungk18** [comment]
@googlebot I fixed it.

File: aio/content/guide/lifecycle-hooks.md [hunk 1]
```diff
@@ -495,7 +495,7 @@ for one turn of the browser's JavaScript cycle, which triggers a new change-dete
 
 #### Write lean hook methods to avoid performance problems
 
-When you run the *AfterView* sample, notice how frequently Angular calls `AfterViewChecked()`$emdash;often when there are no changes of interest.
+When you run the *AfterView* sample, notice how frequently Angular calls `AfterViewChecked()`;often when there are no changes of interest.
```
Comment (atscott) [inline aio/content/guide/lifecycle-hooks.md:498]:
```suggestion
When you run the *AfterView* sample, notice how frequently Angular calls `AfterViewChecked()`—often when there are no changes of interest.
```

**Comment by atscott** [comment]
Hi @trungk18, could you squash the two commits in to one and update the commit message body to be 100 characters long? You can use something like "Fix documentation in the lifecycle hooks guide where `$emdash;` was not being replaced by an actual em dash (—)".
The CLA issue looks like it's because one of your commits was done using a different email (and that email hasn't signed the CLA). Maybe this can be resolved when you squash/amend your commits.

**Comment by trungk18** [comment]
@atscott I pushed the change as your suggestion. For the CLA I think because I pushed with my company email, that's why the bot couldn't realize. I have updated my email before pushing. Hope it will work.

@googlebot I fixed it.

**Comment by trungk18** [comment]
@atscott Sure, let me do it again. Thanks for your advise.

**Pull request closed without merging**